compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -34,10 +34,24 @@ def AMDAIECleanup :
       "mlir::iree_compiler::AMDAIE::createAMDAIECleanupPass()";
 }
 
+def AMDAIEAddLoweringStrategy :
+    Pass<"iree-amdaie-add-lowering-strategy", "mlir::iree_compiler::IREE::HAL::ExecutableVariantOp"> {
+  let summary = "Add lowering strategy configurations to be used";
+  let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEAddLoweringStrategyPass()";
+  let options = [
+    Option<"tilingStrategy", "tiling-strategy", "std::string", /*default=*/"",
+      "The tiling strategy to be used">  
+  ];
+}
+
 def AMDAIELowerExecutableTarget :
     Pass<"iree-amdaie-lower-executable-target", "mlir::iree_compiler::IREE::HAL::ExecutableVariantOp"> {
   let summary = "Perform lowering of executable target using one of the IREE::HAL::DispatchLoweringPassPipeline";
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIELowerExecutableTargetPass()";
+  let options = [
```

Comment:
We can just drop this, and just call the `AIEAddLoweringStrategy` before in our pass-pipeline. So we need to set it only one place.

Reply:
I made this change and added the AIEAddLoweringStrategy pass to the pass pipeline: https://github.com/nod-ai/iree-amd-aie/pull/106/files#diff-42f0d0bb098689f25ee68e8f05ec6c2eaa89ce41a6394d7d99c1d1c912943b38R111